sdk/rust/README.md

```diff
@@ -0,0 +1,83 @@
+# Picovoice SDK for Rust
+
+## Picovoice
+Made in Vancouver, Canada by [Picovoice](https://picovoice.ai)
+
+Picovoice is an end-to-end platform for building voice products on your terms. It enables creating voice experiences
+similar to Alexa and Google. But it entirely runs 100% on-device. Picovoice is
+
+- **Private:** Everything is processed offline. Intrinsically HIPAA and GDPR compliant.
+- **Reliable:** Runs without needing constant connectivity.
+- **Zero Latency:** Edge-first architecture eliminates unpredictable network delay.
+- **Accurate:** Resilient to noise and reverberation. It outperforms cloud-based alternatives by wide margins
+[*](https://github.com/Picovoice/speech-to-intent-benchmark#results).
+- **Cross-Platform:** Design once, deploy anywhere. Build using familiar languages and frameworks.
+
+## Compatibility
+
+- Rust 1.54+
+- Runs on Linux (x86_64), macOS (x86_64), Windows (x86_64), Raspberry Pi, NVIDIA Jetson (Nano), and BeagleBone
+
+## Installation
+First you will need [Rust and Cargo](https://rustup.rs/) installed on your system.
+
+To add the picovoice library into your app, add `picovoice` to your app's `Cargo.toml` manifest:
+```toml
+[dependencies]
+picovoice = "${version}"
+```
+
+## Usage
+
+To create an instance of the engine with default parameters, use the `PicovoiceBuilder` function. You must provide a Porcupine keyword file, a wake word detection callback function, a Rhino context file and a inference callback function. You must then make a call to `init()`.
+
+```rust
+use picovoice::{rhino::RhinoInference, PicovoiceBuilder};
+
+let wake_word_callback = || {
+    // let user know wake word detected
+};
+let inference_callback = |inference: RhinoInference| {
+    if inference.is_understood {
+        let intent = inference.intent.unwrap();
+        let slots = inference.slots;
+        // add code to take action based on inferred intent and slot values
+    } else {
+        // add code to handle unsupported commands
+    }
+};
+
+let mut picovoice = PicovoiceBuilder::new(
+    keyword_path,
+    wake_word_callback,
+    context_path,
+    inference_callback,
+).init().expect("Failed to create picovoice");
+```
+Upon detection of wake word defined by `keyword_path` it starts inferring user's intent from the follow-on voice command within
+the context defined by the file located at `context_path`. `keyword_path` is the absolute path to
+[Porcupine wake word engine](https://github.com/Picovoice/porcupine) keyword file (with `.ppn` suffix).
+`contextPath` is the absolute path to [Rhino Speech-to-Intent engine](https://github.com/Picovoice/rhino) context file
+(with `.rhn` suffix). `wake_word_callback` is invoked upon the detection of wake phrase and `inference_callback` is
+invoked upon completion of follow-on voice command inference.
```

Comment:
Can you also add a section here about overwriting default models and sensitivities?